workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -19,6 +19,8 @@ public class MutableView : IDisposable
         public readonly Dictionary<int, ComponentTranslation> TranslationUnits =
             new Dictionary<int, ComponentTranslation>();
 
+        public readonly HashSet<GameObjectTranslationBase> MonoBehaviourTranslations = new HashSet<GameObjectTranslationBase>();
```

Comment:
Do you mean `GameObjectTranslations`?

Reply:
And ctrl-f `monobehaviourtranslation` in other parts of PR

Reply:
Fixed.